pkgs/development/python-modules/python-hwdata/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hwdata
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "python-hwdata";
+  version = "2.4.1";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

This makes the build use our PEP517 builder.